.circleci/config.yml

```diff
@@ -412,6 +412,10 @@ jobs:
               TEST_FILES=$(circleci tests glob "tests/aws/**/test_*.py" "tests/integration/**/test_*.py" | PYTHONPATH=localstack-core python -m localstack.testing.testselection.scripts.filter_by_test_selection target/testselection/test-selection.txt | circleci tests split --verbose --split-by=timings | tr '\n' ' ')
             fi
             echo $TEST_FILES
+            if [[ -z "$TEST_FILES" ]] ; then
```

Comment:
Did this one here maybe slip into the commit? :)

Reply:
Actually, this is something I want to test out. In the test run linked (since a couple seconds) in the description, we can see the runners fail if not a single test is both selected and assigned to the runner, which leads to a red pipeline.

This is a preliminary, to be discussed change, to prevent that. Would love your input on it! We can also separate it into a separate PR.

Reply:
Oh right, sorry, you mentioned that in the description, sorry! With the increased scope of the test selection this will definitely happen more often. Imho makes sense!
/cc @silv-io @dominikschubert

Reply:
#11105 is the replacement for this!